### PR TITLE
Add Footer

### PR DESCRIPTION
**Pull request by NadiiaLashtun**

### CODE REVIEW

**Comment by MarwanBz** [comment]
Hey @NadiiaLashtun 

thanks for the amzing work, well done!

but I have an error the server didn't run

image

File: src/images.d.ts
Comment (MarwanBz) [inline src/images.d.ts:1]: @NadiiaLashtun could you plz explain these lines of code? what do they do?

**Review by MarwanBz — COMMENTED** [review]
Incredible work

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import React from "react";
+import Footer from "./components/Footer";
 
 function App() {
   return (
-    <div >
-      hello world
+    <div>
```
Comment (parykhan-jameel) [inline src/App.js:6]: a closing tag is missing

File: src/images.d.ts
Comment (parykhan-jameel) [reply src/images.d.ts:1]: it allows you to import svg files as string

**Review by parykhan-jameel — COMMENTED** [review]
all looks good, great work.
but you should've make it in an easier way inside one file

**Pull request merged**